docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -918,4 +918,89 @@ spec:
   platforms: macOS
   tags: compliance, CIS, CIS6.1.4
   platform: darwin
-  contributors: GuillaumeRoss
\ No newline at end of file
+  contributors: GuillaumeRoss
+
+---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure Windows Firewall - Domain Profile is enabled (Windows)
+      query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\DomainProfile\EnableFirewall' AND CAST(data as integer) = 1;
+      description: "Checks for the existence of a GPO that requires the Windows Firewall - Domain Profile is enabled."
+      resolution: "Contact your IT administrator to ensure your Windows system is receiving a GPO that requires that the Windows Firewall - Domain profile is enabled."
```

```suggestion
      resolution: "Contact your IT administrator to ensure your Windows system is receiving a policy that eables the Domain profile of Windows Firewall"
```

Here is some wording I think might be a bit more user friendly - @noahtalerman might have a better suggestion though. I would apply the same to most of the other queries below.

---

I like this suggestion. Guillaume I tweaked your suggested update.

I added the same suggestion to the other queries.